thrift/src/main/thrift/atoms/explainer.thrift

```diff
@@ -1,5 +1,6 @@
 namespace * contentatom.explainer
 namespace java com.gu.contentatom.thrift.atom.explainer
+#@namespace scala com.gu.contentatom.thrift.atom.explainer
```

Comment:
it's an annoying quirk of Thrift that it will complain if it sees a namespace it doesn't understand. So this is a special syntax that scrooge supports which will allow it to be backwards compatible with the original thrift (because Thrift just sees it as a comment), which we need for generating Javascript code for the quiz builder.